tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/remove_non_matching_participant_test.py

```diff
@@ -33,6 +33,8 @@ def setUp(self):
         self.biqquery_job_id = 'job-id-1'
         self.query_results = {'jobReference': {'jobId': self.biqquery_job_id}}
         self.client = mock.MagicMock()
+        type(self.client).project = mock.PropertyMock(
```

Comment:
What does this line do? (I am asking this because `mock` is still my weak point :smiling_face_with_tear: )

Reply:
It mocks the property of a mocked object. In this case, the self.client object.